WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderTableViewCells/ConnectedReaderTableViewCell.xib

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="19455" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="20037" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
```

Comment:
nit: in the Interface Builder, there seems to be some warnings about content size ambiguity that aren't in `trunk`. Is it possible to fix them?

Reply:
Good catch. Fixed it by clicking on some button which apparently automattically fixes that 
af5d5d60d1d84be3f9b0733b9d127821d05e291e